libraries/AP_HAL_ChibiOS/hwdef/JHEMCU-GSF405A/readme.md

```diff
@@ -0,0 +1,93 @@
+# JHEMCU GSF405A AIO Flight Controller
+
+The JHEMCU GSF405A is a 1-2S Whoop-style (25.5mm x 25.5mm) flight controller featuring the STM32F405OG6 MCU, 4in1 5A ESCs, BMP280 barometer, 8MB flash, and an on-board ExpressLRS 2.4GHz receiver. The board weighs 3.3g, making it suitable for small or lightweight Ardupilot builds.
+
+![JHEMCU GSF405A](gsf405a.jpg "JHEMCU GSF405A Top")
+
+https://pyrodrone.com/collections/new-products/products/jhemcu-gsf405a-1s-2s-aio-f4-flight-control-w-5a-esc-elrs-2-4g-rx-25-5-25-5mm
+https://www.racedayquads.com/collections/new-products/products/jhemcu-gsf405a-1-2s-toothpick-whoop-aio-flight-controller-w-8bit-5a-esc-and-elrs-2-4ghz-rx
+
+### Specs:
+**Flight control parameters**
+* MCU: STM32F405OG6
+* Gyroscope/Accelerometer: MPU6000
+* OSD: AT7456E
+* Barometer: BMP280
+* Black box: 8MB
+* I2C: Support
+* BEC: 5V
+* UART: UART1 (ELRS), UART2 (external RC), UART3, UART4, UART6
+* USB: micro usb
+* Size: 25.5*25.5MM M2
+* Receiver: ELRS (CRSF), TBS (CRSF), SBUS, IBUS, DSM2, DSMX
+* Support programmable LED such as WS2812
+* Support buzzer
+* Built-in voltage and current sensors
+* Weight: 3.3 grams
+
+**ESC parameters**
+* Support PWM, Oneshot125, Oneshot42, Multishot, Dshot150, Dshot300, Dshot600
+* Input voltage: 1S-2S Lipo
+* Continuous current: 5A
+* Firmware: BLHELI_S S_H_50_REV16_7.HEX
```

Comment:
link to this firmware would be nice

Reply:
I can add a link to the default BLHeli_S firmware, but using bidirectional DShot would require flashing to a different supported firmware (eg Bluejay or BLHeli_M). I can include links to both and expand upon this under "Motor Output" if it would be helpful.

Reply:
Sorry, I thought this was firmware specific to this board, if it is not, then no need to link.

Reply:
I will update the readme to include a link to the Betaflight target and a link to the official manufacturer page for the board once it is available.

Reply:
I think you should put in the link to whatever works with bdshot - would need testing though. You should probably at least list the config - so S_H_50